app/javascript/src/components/editor/Modals/TranslationKeysModal.svelte

```diff
@@ -60,27 +46,11 @@
             </div>
           {/each}
         </div>
+      {:else}
+        <p class="m-0">No keys</p>
```

Comment:
Perhaps a slightly friendlier empty state would be nice

```suggestion
        <p class="m-0">You have not yet created any keys.</p>
```

Or something like that

Reply:
I was going for less verbosity, but sure :+1: